src/main/java/duke/command/Parser.java

```diff
@@ -65,41 +65,41 @@ public void processLine(String userInput) throws DukeException, ParseException {
             }
             int index = Integer.parseInt(words[1]) - 1;
             Task removed = tasks.delete(index);
-            ui.printDeleteTask(removed, list);
+            return ui.printDeleteTask(removed, list);
 
         } else if (firstWord.equals("find")) {
             String secondWord = words[1];
             ArrayList<Task> listFound = new ArrayList<>();
             tasks.list.stream()
                     .filter(task -> task.getDescription().contains(secondWord))
                     .forEach(listFound::add);
-            ui.printList(listFound);
+            return ui.printList(listFound);
         } else {
             Task task;
             if (firstWord.equals("todo")) {
                 if (words.length < 2 || words[1].equals("")) {
-                    throw new DukeException("☹ OOPS!!! The description of a todo cannot be empty.");
+                    throw new DukeException("OOPS!!! The description of a todo cannot be empty.");
                 }
                 task = new ToDo(userInput.split(" ", 2)[1]);
             } else if (firstWord.equals("deadline")) {
                 if (words.length < 2 || words[1].equals("")) {
-                    throw new DukeException("☹ OOPS!!! The description of a deadline cannot be empty.");
+                    throw new DukeException("OOPS!!! The description of a deadline cannot be empty.");
                 }
                 String description = userInput.split(" ", 2)[1].split(" /", 2)[0];
```

Comment:
Maybe you could process each task in a different function for easier readability